transports/sqs/index.md

```diff
@@ -74,3 +74,20 @@ The SQS transport uses the default [retries and timeouts](http://docs.aws.amazon
 | `ReadWriteTimeout` | 300s          |
 
 NOTE: NServiceBus will perform [immediate](/nservicebus/recoverability/#immediate-retries) and [delayed](/nservicebus/recoverability/#delayed-retries) retries in addition to retries performed internally by the SQS client.
+
+
+## Troubleshooting
+
+### AmazonSQSException: Request is throttled
+
+When sending messages it can happen that the send operation is throttled. When that happens the following exception is logged.
+```
+2017-11-14 23:10:24,314|ERROR|18|NServiceBus.Transports.SQS.MessageDispatcher|Exception from Send.
+Amazon.SQS.AmazonSQSException: Request is throttled. ---> Amazon.Runtime.Internal.HttpErrorResponseException: The remote server returned an error: (403) Forbidden. ---> System.Net.WebException: The remote server returned an error: (403) Forbidden.
+```
+
+When this happens while receiving a message then this can safely be ignored as [recoverability features](/nservicebus/recoverability/) should resolve it. When sending messages outside a message handler those message operations can fail. Sending messages in bulk might increase the chance of being throttled on send to avoid flodding the Amazon SQS infrastructure.
+
+By batching multiple concurrent message operations together in a smart way it is possible to adhere to the throttling constraints. For example send a maximum of 100 messages concurrently as outlined in the [sending large amount of messages guidelines](/nservicebus/handlers/async-handlers.md#concurrency-large-amount-of-concurrent-message-operations).
```

Comment:
> By batching multiple concurrent message operations together in a smart way

What do you mean by "in a smart way"?

Reply:
> it is possible to adhere to the throttling constraints.

Do you mean "avoid throttling"?

Reply:
> For example send a maximum of 100 messages concurrently as outlined in the [sending large amount of messages guidelines]...

Perhaps say "use batching" pointingn to the doco w/o stating a specific number as we don't do that in the referenced number. A batch could be 120 or 200 msgs as well.